### ISSUE TITLE
Add ability to specify VLAN ranges

### ISSUE THREAD

**Issue by v0tti** [ansible-collections/dellemc.enterprise_sonic#61]
##### SUMMARY
I would like to configure VLAN ranges for trunk interfaces. Currently it is only possible to add VLANs in a list, bur this can be quite cumbersome if many (or all) VLANs are needed on an interface.

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME
- sonic_l2_interfaces
- sonic_mclag
- sonic_vlans

##### ADDITIONAL INFORMATION
Instead of:

```yaml
sonic_l2_interfaces:
  - name: PortChannel1
    trunk:
      allowed_vlans:
        - vlan: 101
        - vlan: 103
        - vlan: 104
        - vlan: 105
        - ...
        - vlan: 200
```

I would like the option to configure it somehow like this:

```yaml
sonic_l2_interfaces:
  - name: PortChannel1
    trunk:
      allowed_vlans:
        - vlan_range: 100-200
```

**Comment by kerry-meyer** [comment]
We agree that this functionality would be useful and are adding it to our roadmap for implementation.

**Comment by kerry-meyer** [comment]
We have added the ability to configure VLAN ranges for trunk interfaces with the merging of "https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/149" (playbook support for vlan trunk ranges) to our "main" branch.

This functionality will be included in the upcoming 2.1.0 release of the Dell Ansible Enterprise SONiC resource module collection.
